tests/gemmini/harness_gemmini.py

```diff
@@ -1,17 +1,13 @@
 import os
 import subprocess
 
-import pytest
-
 from exo import compile_procs
 
 GEMMINI_ROOT = os.getenv('GEMMINI_ROOT')
 if GEMMINI_ROOT is None:
     RISCV = os.getenv('RISCV')
     if RISCV is None:
         pass
-        #pytest.skip("skipping gemmini tests; could not find chipyard",
-        #            allow_module_level=True)
```

Comment:
Do we not actually want this here? Seems like the tests ought to skip if chipyard isn't available.

Reply:
It won't compile&run gemmini tests unless RISCV is present, because of this line:
https://github.com/ChezJrk/exo/blob/master/tests/gemmini/harness_gemmini.py#L247

Without RISCV environment, it'll just run the frontend.